### ISSUE TITLE
Running badgekit API Events error

### ISSUE THREAD

**Issue by Bjern** [mozilla/openbadges-badgekit#494]
I have been up all night trying to get badgekit API to run.  The directions are great until actually running the app.  Can someone help with this error...  I could install foreman as there is an error about fully qualified domain name.

* Server: Centos 7
* nodejs: 7.5.0

node app
util.js:962
    throw new TypeError('The super constructor to "inherits" must not ' +
    ^

TypeError: The super constructor to "inherits" must not be null or undefined
    at Object.exports.inherits (util.js:962:11)
    at Object.<anonymous> (/root/badgekit-api/node_modules/spdy/lib/spdy/connection.js:86:6)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/root/badgekit-api/node_modules/spdy/lib/spdy.js:20:19)

**Comment by Bjern** [comment]
ok so i reinstalled badgekit from start.  I also have tried so much I've lost track of whats been done.  Somewhere along the line I installed an old version of node - 6.7.0 so that the events error would go away but it never took affect.  Now it has.  So confusing.

I now have:
* Server: Centos 7
* nodejs: 6.7.0

It seems the server is running since i can do wget localhost:8080 and i get the response index.html.

However I cannot use my server IP address to get any response.  Is there something I need to do to make it listen on the server IP?

I used **pm2 start app** to load my app otherwise it doesn't start.
